static/bidder-info/ccx.yaml

```diff
@@ -8,7 +8,6 @@ capabilities:
       - banner
       - video
 userSync:
-  # Clickonometrics supports user syncing, but requires configuration by the host. contact this
-  # bidder directly at the email address in this file to ask about enabling user sync.
-  supports:
-    - redirect
+  redirect:
+    url: "https://sync.clickonometrics.pl/prebid/set-cookie?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirurl={{.RedirectURL}}"
+    userMacro: "${USER_ID}"
```

Comment:
The redirectURL doesn't seem to be setting the `${USER_ID}` macro. Could you please check? The below is the URL that I am trying to reach to - 
```
https://sync.clickonometrics.pl/prebid/set-cookie?gdpr=&gdpr_consent=&us_privacy=&redirurl=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dccx%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24%7BUSER_ID%7D
```

Reply:
It is working now!